### ISSUE TITLE
Could not load file or assembly Tweetinvi.Core

### ISSUE THREAD

**Issue by sourabhsriom** [linvi/tweetinvi#806]
Hi @linvi , Base on your suggestions on my previous questions, I upgraded my .net version to 4.6.1 and VS to 2017. I was able to compile the code but when I deploy it in my application. I get the following error  when the Twitter related module is called : 

`The test resulted in a failure. The following error occurred: `

`Could not load file or assembly ''Tweetinvi.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7247f14397bf67b9'' or one of its dependencies. The system cannot find the file specified.`

Is this a known issue? The machine running the application has the appropriate .net version installed.

Thank you for your efforts.

**Comment by linvi** [comment]
I am not sure what is going wrong, but it seems that your build does not copy the Tweetinvi dll.

Did you try creating a new project with the same .NET version to see if you can reproduce this issue?

**Comment by blakepell** [comment]
I'm going to share what I think is going on and how to fix it for posterity as I recently had this issue.

I had started writing a wrapper library (.NET Standard 2) around Tweetinvi to make the functionality I need simple to call and in one place.  When I added this to my test project I did so with a project reference and that's why I received the above error.  The project reference didn't bring along that project's dependencies (I would have had to also add those to my test project).  Adding my wrapper library as a NuGet package DID bring along everything and worked as expected.
